src/averageAges.js

```diff
@@ -15,11 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const pAge = person => person.died - person.born;
+  const pSum = (x, y) => x + y;
```

Comment:
Let's create more understandable variable name
```suggestion
  const getPersonAge = person => person.died - person.born;
  const getSum = (x, y) => x + y;
```